medical_appointment/models/medical_appointment.py

```diff
@@ -1,10 +1,11 @@
 # -*- coding: utf-8 -*-
-# © 2016 LasLabs Inc.
-# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+# Copyright 2016 LasLabs Inc., Komit
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
 
-from odoo import fields, models, exceptions, api, _
 from datetime import timedelta
 
+from odoo import fields, models, exceptions, api, _
```

Comment:
Following this logic: https://github.com/OCA/maintainer-tools/blob/master/CONTRIBUTING.md#imports

odoo comes 3rd!